api/src/integrations/epic/epic-orders.js

```diff
@@ -87,10 +87,10 @@ class EpicOrders {
     }
 
     try {
-      const Order = mongoose.model('Order');
+      const Order = mongoose.model(RECORD_TYPE.Order._schemaName);
```

Comment:
I know its not likely to change, but its generally good practice to avoid magic strings.  

It also means that when you reference that model name in multiple places, there is no possible way for them to be inconsistent, because they all reference the single instance of the string from the enum, so can save headaches around misspelling things, etc.

That enum also has other uses, currently it relates the schema name to the UI display name (since they aren't the same).